src/graph.ts

```diff
@@ -115,15 +115,19 @@ export class GraphBuilder {
     // Map of resource logicalId to GraphNode. Allows for easy lookup by logicalId
     cfnElementNodes: Map<string, GraphNode>;
 
+    // If the app has a VpcCidrBlock resource, this will be set to the GraphNode representing it
+    vpcCidrBlockNode?: GraphNode;
```

Comment:
Nit: I'd prefer if we'd save that in the result of `GraphBuilder::build`. E.g. turn this into a `GraphResult` struct that has the necessary params we need.
I wouldn't expect a `Builder` to store state beyond what's needed to build